images/csp_violation_report_service/app/bin/entry.sh

```diff
@@ -0,0 +1,50 @@
+#!/bin/sh
+
+ENV_PATH="/tmp/csp-reports"
+TMP_ENV_FILE="$ENV_PATH/.env"
+
+var_expand() {
+  if [ -z "${1-}" ] || [ $# -ne 1 ]; then
+    printf 'var_expand: expected one argument\n' >&2;
+    return 1;
+  fi
+  eval printf '%s' "\"\${$1?}\"" 2> /dev/null # Variable double substitution to be able to check for variable
+}
+
+
+load_non_existing_envs() {
+  _isComment='^[[:space:]]*#'
+  _isBlank='^[[:space:]]*$'
+  while IFS= read -r line; do
+    if echo "$line" | grep -Eq "$_isComment"; then # Ignore comment line
+      continue
+    fi
+    if echo "$line" | grep -Eq "$_isBlank"; then # Ignore blank line
+      continue
+    fi
+    key=$(echo "$line" | cut -d '=' -f 1)
+    value=$(echo "$line" | cut -d '=' -f 2-)
+
+    if [ -z "$(var_expand "$key")" ]; then # Check if environment variable doesn't exist
+      export "${key}=${value}"
+    fi
+    
+  done < $TMP_ENV_FILE
+}
+
+if [ ! -f "$ENV_PATH/.env" ]; then # Only setup envs once per lambda lifecycle
```

Comment:
nit: could you change this comment so it references ECS task startup instead of lambda